src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import { SearchBar } from "./SearchBar";
```

Comment:
Use named imports: Instead of importing the entire module and then accessing specific exports, you can use named imports to directly import the specific components or functions you need. This can improve code readability and reduce the amount of code you need to write. For example, instead of import { SearchBar } from "./SearchBar";, you can use import SearchBar from "./SearchBar";.